pkg/controller/stacks/stack/stack.go

```diff
@@ -45,7 +46,11 @@ import (
 )
 
 const (
-	stacksFinalizer = "finalizer.stacks.crossplane.io"
+	stacksFinalizer    = "finalizer.stacks.crossplane.io"
+	namespaceMember    = "true"
```

Comment:
nit: when i see these consts later on in the code, they look like variables that could contain any type of value that isn't always super clear from just reading the code.  perhaps a naming scheme like `labelValue*` could make that more clear? e.g., `labelValueActiveParentStack`. 

definitely a nit pick, not blocking.